### PR TITLE
Feat/rbf-native

### LINKED ISSUE #3288: TX details modal + RBF
**Design:** [Zeplin](https://zpl.io/2G87k59)

This update includes: 

- Tx status: Not final (orange), Final (green) 
_Note: Tx status is to be used in incoming TX as well, In future "bump fee" button will be added › CPFP feature_  

- Chained TXs tab 
- Chained TXs warning in RBF (bump fee) form

### PR DESCRIPTION
**Pull request by szymonlesisz**
implementation of "native" RBF (since FW 1.9.4/2.3.5)

depends on [new flow for SignTx message](https://github.com/trezor/connect/pull/734) 

#### T1
Screenshot from 2021-01-06 11-55-24
#### TT
Screenshot from 2021-01-06 11-56-31
Screenshot from 2021-01-06 11-56-42

#### Fixed calculation of RBF new fee rate
Screenshot from 2021-01-06 10-27-31
Screenshot from 2021-01-06 10-27-48

#### New views in RBF form
Screenshot from 2021-02-02 11-44-43
Screenshot from 2021-02-02 11-41-31

close https://github.com/trezor/trezor-suite/issues/3288

### CODE REVIEW

**Comment by slowbackspace** [comment]
design updated, resolving https://github.com/trezor/trezor-suite/issues/3288

**Comment by slowbackspace** [comment]
Another round of design update finished, hopefully this was the last one 🤞

File: packages/suite/src/actions/wallet/sendFormActions.ts [hunk 1]
```diff
@@ -208,7 +208,7 @@ export const signTransaction = (
             transactionInfo: {
                 ...transactionInfo,
                 rbf: formValues.options.includes('bitcoinRBF'),
-                prevTxid: formValues.prevTxid,
+                prevTxid: formValues.rbfParams ? formValues.rbfParams.txid : undefined,
```
Comment (szymonlesisz) [inline packages/suite/src/actions/wallet/sendFormActions.ts:213]: not needed?

File: packages/suite/src/hooks/wallet/form/useCompose.ts [hunk 2]
```diff
@@ -114,6 +113,12 @@ export const useCompose = ({
                         type: 'compose',
                         message: errorMessage as any, // setError types is broken? according to ts it accepts only strings, but object or react component could be used as well...
                     });
+                } else if (defaultFieldRef.current !== 'outputs[0].amount') {
```
Comment (slowbackspace) [inline packages/suite/src/hooks/wallet/form/useCompose.ts:116]: just a nitpick, I would add comment explaining why is this needed, could be really confusing without having full picture.

Comment (szymonlesisz) [reply packages/suite/src/hooks/wallet/form/useCompose.ts:116]: if you expand that block the you will see then whole picture :)
- set error to the field that triggered compose process
- or set error to default field (if it's not an amount, which is default-default)
- or set error to all present amount fields

Comment (szymonlesisz) [reply packages/suite/src/hooks/wallet/form/useCompose.ts:116]: but ok, ill do the explanation here

File: packages/suite/src/actions/wallet/sendFormActions.ts (hunk 1 above)
Comment (szymonlesisz) [reply packages/suite/src/actions/wallet/sendFormActions.ts:213]: needed ;) (used in sendForm actions)
